guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,478 @@
 {
```

Comment:
<p><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.analytics.html#manage-multidimensional-raster" target="_blank">manage_multidimensional_raster</a></code></p><p>is repeated twice in the list of functions added to arcgis.raster.analytics</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Removed one of them!</p>

Reply:
<p>Thanks for looking it over <a href="https://github.com/rhea-j" target="_blank">rhea-j</a>!</p>